src/styles/App.scss

```diff
@@ -56,6 +88,10 @@ h1,h2,h3,h4,h5,h6 {
   margin-bottom: 1.3rem;
 }
 
+b, strong {
+  color: black; // Turn dark-gray text black to make bold stand out more
```

Comment:
Looks great in the blog, risk tracker docs and whitepaper!

It also seems to affect many places where it's not quite appropriate on the calculator itself though (model update; form labels now darker than e.g. the Precaution subheading etc)

is there a way to be more selective about which bold/strong text gets turned to black?

Reply:
Sure! Done.